### DISCUSSION TITLE
Problems with setting up Fastmail

### DISCUSSION THREAD

**Discussion by cubeshit** [TechbeeAT/jtxBoard#1445] [General]
In the compatibility list, it says that Fastmail is supported. So I created a new account and followed all the steps to set it up in DAVx5 and jtxboard. But it seems Fastmail doesn’t work with VTODO and VJOURNAL. In DAVx5 it shows only the calendar symbol and even when I create a new calendar with everything it doesn’t work. 
So I'm wondering if the compatibility list is wrong or if am I making a mistake somewhere.

I also tried it with GMX and there it works.

**Comment by patrickunterwegs** [comment]
Hi, I've also checked fastmail now. It still works properly with collections, that had already journals and tasks set up. But creating a new collection generally fails, no matter if I choose journals only, tasks only, events only or all of them. So either this could be a DAVx5 issue or a problem with fastmail. May I ask you to address this issue on DAVx5?

**Comment by cybrengineer** [comment]
I found that calendar creation only works on DAVx5 and fastmail without a timezone. i.e. scroll to the top and set it to `-`. Also I could only assign one type (Event, task, journal) per calendar.
